### PR TITLE
Improve inline docu by a very small bit

### PR DESCRIPTION
**Pull request by bjoernhaeuser**
@samrocketman 

Nothing major, but some small improvements in the inline documentation.

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-excludedRegions.html [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
     Each exclusion uses regular expression pattern matching, and must be separated by a new line.
-    <p/>
-  <pre>
-	myapp/src/main/web/.*\.html
-	myapp/src/main/web/.*\.jpeg
-	myapp/src/main/web/.*\.gif
-  </pre>
+    <br/>
+    <pre>
+	    myapp/src/main/web/.*\.html
```
Comment (samrocketman) [inline src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-excludedRegions.html]: You're mixing tabs and spaces here.  Please fix all instances of using tabs and switch them to using 4 spaces per tab.  I'm only going to comment this once but this feedback applies to all changes in this PR which contain tabs.

**Comment by bjoernhaeuser** [comment]
@samrocketman I hope I got all the tabs :)

**Comment by bjoernhaeuser** [comment]
I deleted the `.tmp` files. Also we are going to need a proper java style guide. Whenever I start working on a `.java` file my IDE wants to reformat the complete file. :(

**Comment by samrocketman** [comment]
@bjoernhaeuser feel free to merge this.

**Comment by bjoernhaeuser** [comment]
I think we need to remove `continuous-integration/jenkins/pr-merge` as requirec check, its not executed for this PR (assuming its because I created the branch directly here).

Or should I change my workflow to work with a fork?

**Comment by bjoernhaeuser** [comment]
Merging this now, approval was dismissed because I rebased onto master.

**Comment by samrocketman** [comment]
I don't really care whether we work with forks or not.  I added the required status checks so maintainers (including myself) don't accidentally push directly to the branch without reviews and CI verification.

**Pull request merged**